controllers/sensor.js

```diff
@@ -0,0 +1,72 @@
+const Sensor = require("../models/Sensor");
+const Counter = require("../models/Counter");
+
+async function getNextSequenceValue () {
+  try {
+    var sequenceDocument = await Counter.findOneAndUpdate(
+      { _id: "5df7691d1c9d440000bf718d" },
+      { $inc: { sequence_value: 1 } },
+      { new: true });
+  } catch (error) {
+    console.log(error);
+  }
+
+  return sequenceDocument.sequence_value;
+}
```

Comment:
You can return the value inside the `try` block, and avoid using `var`. Eg:
```js
async function getNextSequenceValue () {
  try {
    const sequenceDocument = await Counter.findOneAndUpdate(
      { _id: "5df7691d1c9d440000bf718d" },
      { $inc: { sequence_value: 1 } },
      { new: true });
      return sequenceDocument.sequence_value;
  } catch (error) {
    console.log(error);
  }
}
```

In your way, even if there is an error, you will still do `return sequenceDocument.sequence_value;` which will cause an error because `sequenceDocument` will be undefined.

Reply:
done